### PR TITLE
Create action display view page for approval page sim#29

### PR DESCRIPTION
**Pull request by sruble**
got images uploaded- although its not due to the action method.

### CODE REVIEW

**Review by Sara-webaplication — APPROVED** [review]

**Comment by Sara-webaplication** [comment]
looks good to me

**Review by weberwilliamb — APPROVED** [review]
I didn't catch anything that needs to change.

File: photo-points/Views/Admin/newView.cshtml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿@model photo_points.Models.User
+@addTagHelper *,Microsfot.AspNetCore.Mvc.TagHelpers
```
Comment (RoEric) [inline photo-points/Views/Admin/newView.cshtml]: *Microsoft
Probably needs to be fixed on the other view pages, as well, but overall looks good.

**Review by RoEric — APPROVED** [review]
Nice job.

File: photo-points/Views/Admin/newView.cshtml (hunk 1 above)
Comment (sruble) [reply photo-points/Views/Admin/newView.cshtml]: Thanks Eric!

**Review by MaxIgnatyev — APPROVED** [review]
Looks like it's ok to merge into develop.

**Pull request merged**